sys/src/9/riscv/ether.c

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I put this in amd64/core.json; there are a few devs for which this is done. Just to be clear, though, you want this out of riscv for now just to avoid interface churn? That's totally fine; I'm just trying to get ahead of what'll come down the pike when you're ready to bring up an ethernet interface.